src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -326,11 +282,9 @@ def is_local(self) -> bool:
 
         Returns:
             `True` if the KFP orchestrator is running locally (i.e. in
-            the local k3d cluster managed by ZenML).
+            the local k3d cluster).
         """
-        return self.kubernetes_context == self._get_k3d_kubernetes_context(
-            self.id
-        )
+        return self.config.is_local
```

Comment:
Can we maybe remove this property on the orchestrator now and simply use `self.config.is_local` instead?